workspaces-tests/src/bin/payroll_example.rs

```diff
@@ -0,0 +1,109 @@
+use near_contract_tools::{
+    rbac::{self, Rbac},
+    Rbac,
+};
+use near_sdk::{
+    borsh::{self, BorshDeserialize, BorshSerialize},
+    collections::UnorderedMap,
+    env, ext_contract, near_bindgen, require, AccountId, BorshStorageKey, IntoStorageKey,
+    PanicOnDefault, Promise,
+};
+
+enum PayrollKey {
+    LOG,
+    FEE,
+}
+
+impl IntoStorageKey for PayrollKey {
+    fn into_storage_key(self) -> Vec<u8> {
+        match self {
+            PayrollKey::LOG => b"~pl".to_vec(),
+            PayrollKey::FEE => b"~pf".to_vec(),
+        }
+    }
+}
```

Comment:
You are probably fine to use `BorshStorageKey` for storage keys at the contract level.